pkg/operator/defragcontroller/defragcontroller.go

```diff
@@ -82,29 +93,29 @@ func (c *DefragController) sync(ctx context.Context, syncCtx factory.SyncContext
 }
 
 func (c *DefragController) checkDefrag(ctx context.Context, recorder events.Recorder) error {
-	// Check for existing status.
-	_, status, _, err := c.operatorClient.GetOperatorState()
-	if err != nil {
-		return err
+	disableConfigMap, err := c.configmapLister.ConfigMaps(operatorclient.OperatorNamespace).Get(defragDisableConfigmapName)
+	if err != nil && !k8serror.IsNotFound(err) {
```

Comment:
all good, this can maybe be written a bit less confusing. 

When getting a config map that doesn't 
exist we get a `disableConfigMap = nil, err = k8serror.NotFound`. So that only ensures that we return errors that are happening when the configmap is actually there (connection errors for example).